command.md

```diff
@@ -1320,6 +1333,25 @@ filesystem, this function fails with <a href="#error_code.not_permitted"><code>e
 <ul>
 <li><a name="symlink_at.0"></a> result&lt;_, <a href="#error_code"><a href="#error_code"><code>error-code</code></a></a>&gt;</li>
 </ul>
+<h4><a name="access_at"><code>access-at: func</code></a></h4>
+<p>Check accessibility of a filesystem path.</p>
+<p>Check whether the given filesystem path names an object which is
+readable, writeable, or executable, or whether it exists.</p>
+<p>This does not a guarantee that subsequent accesses will succeed, as
+filesystem permissions may be modified asynchronously by external
+entities.</p>
+<p>Note: This is similar to <code>faccessat</code> with the <code>AT_EACCESS</code> flag in POSIX.</p>
+<h5>Params</h5>
+<ul>
+<li><a name="access_at.this"><code>this</code></a>: <a href="#descriptor"><a href="#descriptor"><code>descriptor</code></a></a></li>
+<li><a name="access_at.path_flags"><a href="#path_flags"><code>path-flags</code></a></a>: <a href="#path_flags"><a href="#path_flags"><code>path-flags</code></a></a></li>
+<li><a name="access_at.path"><code>path</code></a>: <code>string</code></li>
+<li><a name="access_at.ype"><code>ype</code></a>: <a href="#access_type"><a href="#access_type"><code>access-type</code></a></a></li>
```

Comment:
looks like a typo? or is `ype` intended?

Reply:
Oops, that's a typo. I filed https://github.com/WebAssembly/wasi-filesystem/pull/115 to fix it upstream.

Reply:
This is now fixed upstream; I'll merge it in in the next upstream sync.